hare/pb/hare.proto

```diff
@@ -0,0 +1,34 @@
+syntax = "proto3";
```

Comment:
There is no hare.proto anymore (replaced with xdr).
Please make sure you don't push this file